src/App.js

```diff
@@ -15,30 +15,30 @@ import {
 
 
 function App() {
-    const {isOpen, onOpen, onClose} = useDisclosure();
+    const { isOpen, onOpen, onClose } = useDisclosure();
     const btnRef = React.useRef();
 
     return (
         <>
             {/*<Button variantColor="green">Button</Button>*/}
-            <p onClick={onOpen}>Open</p>
+            <p onClick={onOpen}>OpenTest2</p>
```

Comment:
Why don't you try changing this text to another one to see how corrections work?